README.md

```diff
@@ -144,14 +150,13 @@ To configure the LM of your choice, follow these steps:
    ```
    This command will prompt you to enter the LLM API key and model name, ensuring that OpenDevin is tailored to your specific needs.
 
-2. **Manual Config: The Artisanal Touch**
```

Comment:
removed this, since it's not super necessary for getting started. And the `make` output references the `config.toml`